python/grass/jupyter/interactivemap.py

```diff
@@ -347,13 +353,160 @@ def add_layer_control(self, **kwargs):
         else:
             self.layer_control_object = self._ipyleaflet.LayersControl(**kwargs)
 
+    def draw_computational_region(self):
+        """
+        Allow Users to draw the computational region and modify it.
+        """
+        import ipywidgets as widgets
+        from ipyleaflet import Rectangle, WidgetControl
+
+        region_mode_button = widgets.ToggleButton(
+            icon="square-o",
+            value=False,
+            tooltip="Click to add computational region",
+            button_style="info",
+            layout=widgets.Layout(width="33px", margin="0px 0px 0px 0px"),
+        )
```

Comment:
Needs empty description and modified tooltip:
```suggestion
        region_mode_button = widgets.ToggleButton(
            icon="square-o",
            description="",
            value=False,
            tooltip="Click to show computational region",
            button_style="info",
            layout=widgets.Layout(width="33px", margin="0px 0px 0px 0px"),
        )
```